file-tests/should-futil/fixed-point-constant.expect

```diff
@@ -13,25 +13,21 @@ component main() -> () {
     fp_const3 = std_const(15,19968);
   }
   wires {
-    group let0<"static"=1> {
+    static group <1> let0 {
       a_0.in = fp_const0.out;
       a_0.write_en = 1'd1;
-      let0[done] = a_0.done;
     }
-    group let1<"static"=1> {
+    static group <1> let1 {
```

Comment:
I'm confused, I thought the latency attribute comes after the name? Also, can we get rid of the space between the name and the latency?

Reply:
ah yeah my bad. I'll change it.